Tools/autotest/sim_vehicle.py

```diff
@@ -833,6 +833,7 @@ def start_vehicle(binary, opts, stuff, spawns=None):
         cmd.append("--start-time=%d" % start_time_UTC)
 
     cmd.append("--sim-address=%s" % cmd_opts.sim_address)
+    cmd.append("--enable-fgview")
```

Comment:
Should we make this a command-line option to avoid unnecessary overhead in 99.99% of sim_vehicle.py invocations?

Reply:
yes, we should